index.bs

```diff
@@ -283,10 +283,13 @@ A {{Sanitizer}} has an associated <dfn for="Sanitizer">configuration</dfn>, a {{
 The <dfn for="Sanitizer" export>constructor</dfn>(|configuration|)
 method steps are:
 
+1. Let |allowCommentsAndDataAttributes| be true.
 1. If |configuration| is a {{SanitizerPresets}} [=string=], then:
     1. [=Assert=]: |configuration| [=is=] {{SanitizerPresets/default}}.
     1. Set |configuration| to the [=built-in safe default configuration=].
-1. Let |valid| be the return value of [=set a configuration|setting=] |configuration| on [=this=].
+    2. Set |allowCommentsAndDataAttributes| to false.
```

Comment:
I wonder if it wouldn't be simpler to just add "comments" and "dataAttributes" keys to the built-in default config in §3.4 Builtins. Then the rest would just do the right thing.

Reply:
We like this approach better, because it puts the whole (default) config into one place. I am going to do this.